activerecord/lib/active_record/log_subscriber.rb

```diff
@@ -43,8 +43,10 @@ def sql(event)
         binds.prepend("  ")
       end
 
-      name = colorize_payload_name(name, payload[:name])
-      sql  = color(sql, sql_color(sql), true)
+      if colorize_logging
+        name = colorize_payload_name(name, payload[:name])
```

Comment:
Do this need to be inside the `if`. `color` will take care of it.

Reply:
@rafaelfranca I did the necessary changes for it. Please have a look. Thanks